.github/workflows/docker.yml

```diff
@@ -68,24 +71,25 @@ jobs:
         steps:
             - uses: actions/checkout@v3
             - name: Update Rust
+              if: env.SELF_HOSTED_RUNNERS == false
               run: rustup update stable
             - name: Dockerhub login
               run: |
                   echo "${DOCKER_PASSWORD}" | docker login --username ${DOCKER_USERNAME} --password-stdin
             - name: Cross build Lighthouse binary
+              if: env.SELF_HOSTED_RUNNERS == false
```

Comment:
Unsure about whether this should be here. Do we install dependencies like this on the server itself?

Reply:
If I'm not wrong, the builds are executed in fresh containers on the self hosted runners, and the containers will be removed after the builds, so it should be ok to install them. The reason we had `env.SELF_HOSTED_RUNNERS == false` on some of the installation steps in `test-suite` is because they actually fails, probably due to not having root permissions. Relevant comments [here](https://github.com/sigp/lighthouse/pull/4506/files#r1270159687).

Reply:
Ah thanks! I'll scap em then